flag-app/flag-app-project/src/CountriesContextProvider.tsx

```diff
@@ -4,12 +4,24 @@ import { useApiData } from "./hooks";
 
 const CountriesContextProvider: FC<PropsWithChildren> = ({ children }) => {
   const [searchInput, setSearchInput] = useState("");
-  const { data: countries, error, isLoading } = useApiData(searchInput);
+  const [filterValue, setFilterValue] = useState("Filter by Region");
+  const {
+    data: countries,
+    error,
+    isLoading,
+  } = useApiData(
+    searchInput,
+    filterValue === "Filter by Region" ? "" : filterValue
```

Comment:
Here it's also better to compare by "" because this is confusing - it's not supposed to belong in `filterValue`